Tasks/Ryabov/task5/task5.py

```diff
@@ -0,0 +1,55 @@
+
+
+
+def check_str(s):
+    d = {'up' : 0, 'lo' : 0}
+    if s == '':
+        return print('string is empty')
+    for i in s:
+        if i.isupper():
+            d['up']+=1  
+        elif i.islower():
+            d['lo']+=1   
+        else: pass
+    return print ("Original String : ", s, "No. of Upper case characters : ", d["up"], "No. of Lower case Characters : ", d["lo"])
+
+check_str('The quick Brown Fox')
+
+
+def is_prime(x):
+    if x % 2 == 0:
+        return x == 2 
+    y = 3
+    while y*y <= x:
+        y +=1
+    return y*y > x
+
+print(is_prime(8))
+
+def get_rangers(l):
+    x = str(l[0])
+    
+    for i in range(len(l)-1):
+        
+        
+        if l[i]+1 == l[i+1]:
+            if x[-1] !='-':
+                x +='-'
+            else:
+                pass
+
+        elif l[i]+1 != l[i+1]:
+            x += f'{l[i]}, {l[i+1]}'
```

Comment:
the problem is in that line, you add a new elem to the string and get something like "1-4, 7-8, 10", and then you do the same thing for the very same element but now it represents the end of the range, so you receive "1-4, 7-8, 1010, 12", you can fix this by a simple check before this addition